sdk/cosmos/azure-cosmos/azure/cosmos/aio/scripts.py

```diff
@@ -49,21 +50,27 @@ class ScriptsProxy(object):
     :func:`ContainerProxy.scripts` attribute.
     """
 
-    def __init__(self, container, client_connection, container_link):
-        # type: (ContainerProxy, CosmosClientConnection, str) -> None
+    def __init__(
+            self,
+            container: _ContainerProxy,
+            client_connection: _CosmosClientConnection,
+            container_link: str
+    ) -> None:
         self.client_connection = client_connection
         self.container_link = container_link
         self.container_proxy = container
 
-    def _get_resource_link(self, script_or_id, typ):
-        # type: (Union[Dict[str, Any], str], str) -> str
+    def _get_resource_link(self, script_or_id: Union[Dict[str, Any], str], typ: str) -> str:
         if isinstance(script_or_id, str):
             return u"{}/{}/{}".format(self.container_link, typ, script_or_id)
         return script_or_id["_self"]
 
     @distributed_trace
-    def list_stored_procedures(self, max_item_count=None, **kwargs):
-        # type: (Optional[int], Any) -> AsyncItemPaged[Dict[str, Any]]
+    def list_stored_procedures(
+            self,
+            max_item_count: Optional[int] = None,
```

Comment:
Curious why you left this `max_item_count` alone instead of making kword-only

Reply:
oops, because I missed it 🤦 thank you for catching this!